src/views/FineView.vue

```diff
@@ -254,10 +264,22 @@ const openHandoutEvent = async (eventId: number) => {
 };
 
 const notifyUsers = async () => {
+  isNotifying.value = true;
   await apiService.debtor.notifyAboutFutureFines({
     userIds: selection.value.map((item: any) => item.id),
     referenceDate: secondDate.value?.toISOString() || new Date().toISOString()
-  });
+  })
```

Comment:
Why are you both awaiting, and using .then()? https://dev.to/masteringjs/using-then-vs-async-await-in-javascript-2pma